service/src/test/java/com/commercetools/util/HttpRequestUtilParallelTest.java

```diff
@@ -24,52 +27,108 @@
  */
 public class HttpRequestUtilParallelTest {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(HttpRequestUtilParallelTest.class);
+
     // try to make 200 simultaneous requests in 200 threads
     private final int nThreads = CONNECTION_MAX_TOTAL;
-    private final int requests = CONNECTION_MAX_TOTAL;
+    private final int nRequests = CONNECTION_MAX_TOTAL;
 
     @Test
     public void executeGetRequest_shouldBeParalleled() throws Exception {
-        makeParallelRequests(nThreads, requests, () -> {
-            try {
-                final HttpResponse httpResponse = executeGetRequest("http://httpbin.org/get");
-                assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(HttpStatus.SC_OK);
-                return 1;
-            } catch (IOException e) {
-                throw new RuntimeException("Request exception", e);
-            }
-        });
+        makeAndAssertParallelRequests(nThreads, nRequests, createRequestsSupplier(
+                () -> executeGetRequest(HTTP_HTTPBIN_ORG_GET),
+                // in get requests asserted only response status
+                httpResponse -> assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(HttpStatus.SC_OK)));
```

Comment:
It is a bit hard to read. Could we put it in some variable separately?
```
requestSupplier = createRequestsSupplier(
                () -> executeGetRequest(HTTP_HTTPBIN_ORG_GET),
                // in get requests asserted only response status
                httpResponse -> assertThat(httpResponse.getStatusLine().getStatusCode()).isEqualTo(HttpStatus.SC_OK));
makeAndAssertParallelRequests(nThreads, nRequests, requestSupplier);
```

Reply:
maybe like this: 32045660